frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -73,6 +75,12 @@ export default function SurveyEdit(): JSX.Element {
         setSelectedQuestion(newIndex)
     }
 
+    useEffect(() => {
+        if (survey.events && survey.events.length > 0) {
+            setSurveyValue('conditions', { url: '' })
```

Comment:
why do we remove the `url` and any other checks on conditions when events change?